src/cloud-api-adaptor/gcp/README.md

```diff
@@ -121,6 +135,7 @@ kubectl apply -k "github.com/confidential-containers/operator/config/samples/ccr
 Build and push docker image to a registry:
 
 ```bash
+cd ../ # make sure you are outside the podvm-mkosi directory
```

Comment:
Thanks @beraldoleal. Would you also consider publishing the instructions here - https://confidentialcontainers.org?

Reply:
Hi @bpradipt , sure.

I'm just ruling out some potential issues and I will do that.

Reply:
I was going to suggest linking to https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/docs/DEVELOPMENT.md but then I realized that doc is outdated... :(